controler/usersControler.js

```diff
@@ -0,0 +1,43 @@
+const peopleModel = require("../models/People");
+const getUsers = (req, res, next) => {
+  res.render("users");
+};
+
+const createUser = async (req, res, next) => {
+  try {
+    const peopleInfo = await peopleModel.find();
+    if(peopleInfo== []){
+            console.log("NO data found!");
+            return res.status(404).json({
+                status: 404,
+                message: 'NO data found!'
+        })
+    }
+    peopleInfo.map((data) => {
+      if (data.email == req.body.email) {
+        console.log("The user is already exist!");
+        res.status(409).json({
+          status: 409,
+          message: "The user is already exist!",
+        });
+      }
+    });
+    await peopleModel.create(req.body, (err) => {
```

Comment:
This part is a little bit confusing to me since we are using `async/await` then what would be the reason to use callback function. Try catch should be bubble up the error. I think refactor this is a good way like either use `async/await` or `callback` or `promises`